demos/tests/cases.py

```diff
@@ -672,6 +672,25 @@ def single_option_cases(key, *args):
         }),
     )),
 
+    PythonDemo(name='classification_demo',
+            device_keys=['-d'],
+            test_cases=combine_cases(
+        TestCase(options={
+            '--no_show': None,
+            '-i': DataDirectoryOrigFileNamesArg('classification'),
+            '--labels': str(OMZ_DIR / 'data/dataset_classes/imagenet_2012.txt')}),
```

Comment:
The `classification_benchmark_demo` is tested with `-gt` key, maybe in this demo it is also worth to test?

Reply:
-gt key and associated code was removed.